src/stackanalysismodule.ts

```diff
@@ -68,9 +67,12 @@ export module stackanalysismodule {
               const options = {};
               options['uri'] = `${
                 Apiendpoint.STACK_API_URL
-              }stack-analyses/${respId}?user_key=${
+                }stack-analyses/${respId}?user_key=${
                 Apiendpoint.STACK_API_USER_KEY
-              }`;
+                }`;
+              options['headers'] = {
+                'x-3scale-account-secret': 'not-set'
+              };
```

Comment:
`x-3scale-account-secret` is no longer needed as we are dealing with staging data
```suggestion
```

Reply:
Are we not making these changes for production data?
For the release of free-tier?

Reply:
It was meant for devcluster, not required for staging or prod.